libs/ledger-live-common/src/account/importAccounts.test.ts

```diff
@@ -181,13 +181,21 @@ describe("importAccountsMakeItems", () => {
         mode: "update",
       },
     ]);
+    const syncResult = {
+      failed: {},
+      synchronized: {},
+    };
+    result.accounts.forEach((a) => {
+      syncResult.synchronized[a.id] = a;
+    });
```

Comment:
the test is defined inside `test.skip("should migrate ethereum accounts", () => ...` (line 15): i think all of it is skipped and never tested

Reply:
good catch :o

Reply:
(however, that's not the added test, this was the historical test that I adapted, but i'll try to update it then)